inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -28,49 +29,59 @@
       {{ record_collection_heading() }}
     </div>
     <div id="record-title">
-      {{ record_title() }}
+      {{ render_record_title() }}
     </div>
     <div id="record-authors">
       {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
     </div>
+    <div id="record-year">
```

Comment:
This date seems kinda out of place(?)
<img width="875" alt="capture d ecran 2015-12-14 a 12 22 44" src="https://cloud.githubusercontent.com/assets/2742143/11779741/6e0045f6-a25d-11e5-9bd3-0b499e00ae35.png">